tutor/src/routes.coffee

```diff
@@ -1,8 +1,7 @@
 async = require './helpers/webpack-async-loader'
 
 getCourseListing = ->
-  {CourseListing} = require './components/course-listing'
-  CourseListing
+  require './components/course-listing'
```

Comment:
is [this](https://github.com/openstax/tutor-js/blob/master/tutor/src/components/course-listing.cjsx#L57) meant to be pulled out of an object then?  and also, since we have a [course-listing directory](https://github.com/openstax/tutor-js/blob/master/tutor/src/components/course-listing), should we move [course-listing.cjsx](https://github.com/openstax/tutor-js/blob/master/tutor/src/components/course-listing) into that folder as `index.cjsx`?